components/Author/Author.module.scss

```diff
@@ -0,0 +1,34 @@
+.author {
+  align-content: center;
+  align-items: center;
+  justify-content: center;
+
+  &--vertical {
```

Comment:
I might be wrong but I think we prefer camel-casing for classes with CSS modules but @andrewbarnesweb will know best! 🙂

Reply:
Yeah no worries, went with the coding standards you use at work. If Barnes agrees will change.

Reply:
Agree with Mark 🙂

Reply:
Can you give me some examples please? So how do we differentiate between elements and modifiers with camel casing?

Reply:
With CSS Modules, you just ignore BEM completely. Since BEM tries to solve a problem in standard CSS that CSS Modules deals with out of the box. Making your CSS scoped to a component.

So we can just use names pretty primitively and obvious.

Depending on what you're doing, you can use `composes` too. But I'm still relatively new to CSS Modules, and haven't used that much 🙂  https://github.com/css-modules/css-modules#composition

You also might get linting errors to start with, since I did when I used it in a repo at work. Composes wasn't recognised as a term.

```css
.author {
  align-content: center;  
  align-items: center;  
  display: flex;
  justify-content: center;
}

.vertical {
  composes: author;
  flex-direction: column;
}

.horizontal {
  composes: author;
  flex-direction: row;   
}
```

And then just pull in `vertical` or `horizontal` in the component.

---

Off topic, I'm not sure what browsers we are supporting. But flex gap is fairly new, so if we are happy to just support new browsers, then that's all good 👍🏻

Reply:
I've started using `gap` these days but yeah, support isn't 100%, it's in all the biggies (https://caniuse.com/?search=gap) so unless we're expecting many users in IE then I think we'll be ok! 🙂

Reply:
All actioned and ready for review